crates/query-engine/sql/src/sql/ast.rs

```diff
@@ -91,19 +91,11 @@ pub struct Delete {
 pub struct Update {
     pub schema: SchemaName,
     pub table: TableName,
-    pub alias: TableAlias,
-    pub set: BTreeMap<ColumnName, UpdateExpression>,
+    pub set: BTreeMap<ColumnName, InsertExpression>,
```

Comment:
It's nice to be able to reuse parts, but we should consider calling it something other than `InsertExpression` if it's used for both inserts and updates.

Reply:
Renamed to `MutationValueExpression`. Thanks!